tests/unit/IonTextWriterTest.writeSymbolToken.js

```diff
@@ -0,0 +1,70 @@
+define(
+    function(require) {
+        const registerSuite = require('intern!object');
+        const assert = require('intern/chai!assert');
+        const ion = require('dist/amd/es6/Ion');
+        const iontext = require('dist/amd/es6/IonText');
+
+        registerSuite({
+            name: 'Text Writer.writeSymbolToken',
+
+            false:  () => verifyQuotesAdded('false'),
+            nan:    () => verifyQuotesAdded('nan'),
+            null:   () => verifyQuotesAdded('null'),
+            true:   () => verifyQuotesAdded('true'),
```

Comment:
Might as well be exhaustive with the keywords and add `+inf` and `-inf`.

Reply:
done, thanks :)